dev/packages/beats/auditd-0.0.1/dataset/log/fields/package-fields.yml

```diff
@@ -1,112 +1,62 @@
-      type: group
-      fields:
-        - name: id
-          type: keyword
-          description: >
-            One or multiple unique identifiers of the user.
-        - name: name
-          type: keyword
-          example: albert
-          description: >
-            Short name or login of the user.
-        - name: group.id
-          type: keyword
-          description: >
-            Unique identifier for the group on the system/platform.
-        - name: group.name
-          type: keyword
-          description: >
-                Name of the group.
-    - name: effective
-      type: group
-      fields:
-        - name: id
-          type: keyword
-          description: >
-            One or multiple unique identifiers of the user.
-        - name: name
-          type: keyword
-          example: albert
-          description: >
-            Short name or login of the user.
-        - name: group.id
-          type: keyword
-          description: >
-            Unique identifier for the group on the system/platform.
-        - name: group.name
-          type: keyword
-          description: >
-                Name of the group.
-    - name: filesystem
-      type: group
-      fields:
-        - name: id
-          type: keyword
-          description: >
-            One or multiple unique identifiers of the user.
-        - name: name
-          type: keyword
-          example: albert
-          description: >
-            Short name or login of the user.
-        - name: group.id
-          type: keyword
-          description: >
-            Unique identifier for the group on the system/platform.
-        - name: group.name
-          type: keyword
-          description: >
-                Name of the group.
-    - name: owner
-      type: group
-      fields:
-        - name: id
-          type: keyword
-          description: >
-            One or multiple unique identifiers of the user.
-        - name: name
-          type: keyword
-          example: albert
-          description: >
-            Short name or login of the user.
-        - name: group.id
-          type: keyword
-          description: >
-            Unique identifier for the group on the system/platform.
-        - name: group.name
-          type: keyword
-          description: >
-                Name of the group.
-    - name: saved
-      type: group
-      fields:
-        - name: id
-          type: keyword
-          description: >
-            One or multiple unique identifiers of the user.
-        - name: name
-          type: keyword
-          example: albert
-          description: >
-            Short name or login of the user.
-        - name: group.id
-          type: keyword
-          description: >
-            Unique identifier for the group on the system/platform.
-        - name: group.name
-          type: keyword
-          description: >
-                Name of the group.
+  - name: terminal
+    type: keyword
```

Comment:
The description is now removed here and this is not a group. Not sure if this is on purpose?

Reply:
> Did you by chance remove for descriptions then you wanted? Only the group ones should be removed.

Misunderstanding. I corrected the source code. Only group fields have stripped description now.

> This only works for modules where we actually migrated fields to ECS and did not use ECS from day one. Newer modules have used ECS from day one so you can't find the ECS fields based on a migration flag.

I adjusted the PR:
1. If the field is an alias and has `migration: true`, then skip it.
2. If the field is an alias and the path is same as ECS, then skip it.